package/MDAnalysis/core/topology.py

```diff
@@ -516,6 +516,26 @@ def add_TopologyAttr(self, topologyattr):
         topologyattr.top = self
         self.__setattr__(topologyattr.attrname, topologyattr)
 
+    def del_TopologyAttr(self, topologyattr):
+        """Remove a TopologyAttr from the Topology.
+
+        If it is not present, nothing happens.
+
+        Parameters
+        ----------
+        topologyattr : TopologyAttr
+
+        .. versionadded:: 2.0.0
+        """
+        try:
+            self.__delattr__(topologyattr.attrname)
+        except AttributeError:
+            pass
```

Comment:
Any chance we can have tests that covers this and the other uncovered section below?